extensions/plans.yml

```diff
@@ -150,67 +150,87 @@ postgres:
     volume_size: 100
     multi_az: true
     instance: db.t3.small
-    storage_type: io1
+    storage_type: io2
     iops: 40000
 
-  # 2v CPU, 8GB RAM, 100GB Storage
+  # # 2v CPU, 4GB RAM, 100GB Storage
   medium:
+    volume_size: 100
+    multi_az: false
+    instance: db.t3.medium
+
+  # # 2v CPU, 4GB RAM, 100GB Storage, multi AZ
+  medium-ha:
+    volume_size: 100
+    multi_az: true
+    instance: db.t3.medium
+
+  # 2v CPU, 4GB RAM, 100GB Storage, multi AZ
+  medium-high-io:
+    volume_size: 100
+    multi_az: true
+    instance: db.t3.medium
+    storage_type: io2
+    iops: 40000
+
+  # 2v CPU, 8GB RAM, 100GB Storage
+  large:
     volume_size: 100
     multi_az: false
     instance: db.m5.large
 
   # 2v CPU, 8GB RAM, 100GB Storage, multi AZ
-  medium-ha:
+  large-ha:
     volume_size: 100
     multi_az: true
     instance: db.m5.large
 
   # 2v CPU, 8GB RAM, 100GB Storage, multi AZ
-  medium-high-io:
+  large-high-io:
     volume_size: 100
     multi_az: true
     instance: db.m5.large
-    storage_type: io1
+    storage_type: io2
     iops: 40000
 
   # 8v CPU, 32GB RAM, 564GB Storage
-  large:
+  2x-large:
     volume_size: 564
     multi_az: false
     instance: db.m5.2xlarge
```

Comment:
What happened to `xlarge`? We jumped from `large` to `2xlarge` here. There isn't an option for 16 GB RAM.

Reply:
Good spot. I got confused, thinking `2xlarge` was `2 x large`. Have added in it.

Reply:
Fair point. Will confirm with @WillGibson about what volume size is appropriate for xlarge instances as the jump from large to 2xlarge is 100 to 564

Reply:
As the t-shirt sizes are more about processing and RAM, I was thinking start them all small and people can wind them up as needed.

Reply:
All plans have been changed to have a volume_size of 100gb as default. These can be modified in the <application>-deploy repo when an application is being configured for deployment.